packages/shared/__tests__/ReactErrorUtils-test.internal.js

```diff
@@ -55,6 +55,34 @@ describe('ReactErrorUtils', () => {
     expect(context.didCall).toBe(true);
   });
 
+  if (!__DEV__ && window.hasOwnProperty('event')) {
```

Comment:
So is this always `false`? Then this shouldn't be a test at all if it never executes.

You could, however, "polyfill" `window.event` in jsdom yourself somehow.